mantle/cmd/ore/gcloud/upload.go

```diff
@@ -50,11 +51,17 @@ var (
 
 func init() {
 	cmdUpload.Flags().StringVar(&uploadBucket, "bucket", "", "gs://bucket/prefix/")
-	cmdUpload.MarkFlagRequired("bucket")
+	if err := cmdUpload.MarkFlagRequired("bucket"); err != nil {
+		plog.Error(err)
```

Comment:
I think https://github.com/coreos/coreos-assembler/pull/1866#discussion_r524372977 applies here and below too.

Reply:
Thank you for review. Updated.